cms/static/js/utils/date_utils.js

```diff
@@ -60,10 +60,10 @@ function($, date, TriggerChangeEventOnEnter) {
             time = $(timepickerInput).timepicker('getTime');
         }
         if (date && time) {
-            return new Date(Date.UTC(
+            return new Date(
```

Comment:
@c4milo This change save date and time in tz of user and with this commit becef53cbcbe2839edbdbe2f2122e686d81793dd is set UTC.

Reply:
Shouldn’t it be the other way around? Saving in UTC and displaying or rendering with user’s TZ applied.

Reply:
@c4milo If the date is saved in UTC it has no effect, because in LMS subtract the hours according to the tz

Reply:
Oh, what I’m suggesting is to render the time in Studio using the timezone of the instructor.

Reply:
It also has no effect because it is still stored in UTC

Reply:
What do you mean? Dates are usually stored in UTC, it is a good practice. The difference is how you display the information to the users. If instructor and student are on the same timezone and we display dates in their timezones, they should see the same date and time.